.github/workflows/check-upstream-upgrade.yml

```diff
@@ -0,0 +1,40 @@
+# WARNING: This file was build based on https://github.com/pulumi/ci-mgmt
+# but it is currently not in sync. Any update must be done manually
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+jobs:
+  check_upgrade_provider:
+    name: Check for upstream provider upgrades
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+    - name: Setup tools
+      uses: ./.github/actions/setup-tools
+      with:
+        tools: go
+    - name: Install upgrade-provider
+      run: go install github.com/pulumi/upgrade-provider@main
+      shell: bash
+    - name: "Set up git identity: name"
+      run: git config --global user.name equinix-bot
+      shell: bash
+    - name: "Set up git identity: email"
+      run: git config --global user.email bot@equinix.com
```

Comment:
This is not a thing. Should we be using the default GitHub name and email for commit messages?

Reply:
good point